src/components/Home.js

```diff
@@ -114,6 +115,7 @@ const Home = () => {
           </Select>
         </div>
       </header>
+      {issuesList && menu === 1 ? <Issues /> : menu === 2 ? <Issues /> : null}

```

Comment:
putting the issue component here will change the order of components supposed to load. first home then labels and then issue components should appear.
![Screenshot 2022-10-02 at 21-08-03 Issue Finder](https://user-images.githubusercontent.com/17983225/193462618-157b1fea-0c39-4cbd-8624-04c8bf52dbd4.png)

whatever we change we must preserve this order 

1. Home,
2. Label,
3. Issue Components